src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -48,5 +48,6 @@
   </Assets>
   <Prerequisites>
     <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.DiaSymReader.Native" Version="[15.0,16.0)" DisplayName="Windows PDB reader/writer" />
```

Comment:
Hmm.  I guess that's true since we don't ahve a pure tools package for the raw command line compilers.  The Microsoft.Net.Compilers package is a combination of command line + MSBuild.  If we every separated out it though then we would be able to yank these correct?